_posts/2019-02-27-partner-attributes.markdown

```diff
@@ -0,0 +1,41 @@
+---
+permalink: v1/partner_attributes
+layout: sidebar
+title: Partner Attributes
+---
+
+# Partner Attributes
+
+## Attributes
+
+| Attribute                     | Type              | Read-Only | Optional | Description
+| :----------                   |:-------------     |:---------:|:--------:|:-------------
+| `company_id`                  | String            |     X     |    X     | unique identifier of this company in your system. (Not to be confused with Gusto's company id)
+| `user_id`                     | String            |     X     |    X     | unique identifier of the company's primary payroll admin in your system.  
+| `grants_rev_share`            | Boolean           |     X     |    X     | 
```

Comment:
Unsure about this structure.
- [ ] Is user information needed here?
- [ ] Is a simple flag for rev sharing enough? Partner agnostic enough?

Reply:
I don't know enough to answer this.  @jgrahamthomas : can you answer this before accepting?

Reply:
I think we can do without user information (e.g. identifying the primary payroll admin). It was a suggestion by Eric M. (Xero) to help them map the Gusto company data to Xero data. But I think they can do without.